python/doc/examples/data_analysis/distribution_fitting/plot_estimate_gev_fremantle.py

```diff
@@ -1,167 +1,470 @@
 """
```

Comment:
The GEV examples are scattered among all the other examples and gathering them would be nice for those who want to have  all the examples related to extreme value treatment quickly. But creating a "GEV " section inside the 'Distribution fitting' section would be strange because it would break the balance of the section. We could gather them at the end of the example list: but I did not find the file where the list is specified.....